Makefile.pre.in

```diff
@@ -1933,6 +1934,7 @@ regen-typeslots:
 		$(srcdir)/Objects/typeslots.inc.new
 	$(UPDATE_FILE) $(srcdir)/Objects/typeslots.inc $(srcdir)/Objects/typeslots.inc.new
 
+
```

Comment:
Should probably remove the added blank line

Reply:
Yeah :) I probably should have put this back in draft, but I added a dummy commit that actually uses the macros just to make sure things build on windows. Will remove the line and the commit shortly.